IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_DistributionSystemTypeOverheadContactlineSystem/Documentation.md

```diff
@@ -0,0 +1 @@
+Properties of a overhead contact line system. The property set is associated with the predefined type OVERHEAD_CONTACT_LINE_SYSTEM of _IfcDistributionSystem_.
```

Comment:
```suggestion
Properties of an overhead contact line system. The property set is associated with the predefined type OVERHEAD_CONTACT_LINE_SYSTEM of _IfcDistributionSystem_.
```

Reply:
Updated: https://github.com/bSI-InfraRoom/IFC-Specification/pull/190/commits/be731fb1169a876200741df2c81db4c91e730337

Reply:
Don't see this change yet.